ChangeLog.d/psa_hash_without_init.txt

```diff
@@ -0,0 +1,4 @@
+Features
+   * Document that PSA hash operations don't require psa_crypto_init(). This
+     has always been possible, but was not previously explicitly stated. This
```

Comment:
Except when there's a driver that requires initialization, but that's a [preexisting bug](https://github.com/Mbed-TLS/mbedtls/issues/6469). Making this promise makes the bug more pressing though, so it would be good to fix it.

Reply:
I disagree with the reasoning here. As you say, it's technically a new feature, so I'm not sure how 6469 can be considered a pre-existing bug.

Regardless, we really shouldn't write things that we know to be false, so we need to either mention 6469 as temporary limitation that will be lifted in the future, or (much preferred) solve 6469 before we merge this.

Reply:
I am taking advantage of the fact that `MBEDTLS_PSA_CRYPTO_DRIVERS` is still considered experimental to declare that this a feature, not a bug.

It's still not clear to me that lifting the limitation is worth the effort (not in terms of coding, but in terms of impact).